staging/src/k8s.io/component-base/metrics/prometheus/restclient/metrics.go

```diff
@@ -41,6 +41,18 @@ var (
 		[]string{"verb", "host"},
 	)
 
+	// resolverLatency is a Prometheus Histogram metric type partitioned by
+	// "network", and "address" labels. It is used for the rest client DNS resolver latency metrics.
```

Comment:
I see that it's consistent with the comment on `requestLatency`, but isn't this all self-evident? The comment (and now that I'm looking closer, the help text too) has already gone out of sync with the actual set of labels.

Reply:
Done